apps/aens/src/aens_utils.erl

```diff
@@ -59,12 +60,15 @@ check_claimed_status(Name) ->
         revoked -> {error, name_revoked}
     end.
 
+name_parts(Name) ->
+    binary:split(Name, ?LABEL_SEPARATOR, [global, trim]).
+
 validate_name(Name) ->
-    case binary:split(Name, ?LABEL_SEPARATOR, [global, trim]) of
+    case name_parts(Name) of
```

Comment:
This isn't optimal - better to know what height you are at and reject `.test`/`.aet` accordingly...

Reply:
ticket created for after Lima release: https://www.pivotaltracker.com/story/show/168246084